src/bin/pg_autoctl/monitor.c

```diff
@@ -1886,6 +1928,13 @@ monitor_listen_loop(Monitor *monitor)
 			log_warn("Re-establishing connection. We might miss notifications.");
 			pgsql_finish(&(monitor->pgsql));
 
+			if (!ensure_monitor_pg_running(monitor, mconfig))
+			{
+				log_warn("unable to start monitor, will re-attempt in %d seconds", PG_AUTOCTL_MONITOR_SLEEP_TIME);
```

Comment:
This warning message is confusing, we're not starting the monitor but Postgres. Our users are already confused about what is `pg_autoctl` and where Postgres is running and what's the role of that thing we call a “monitor”, so I think we should be careful here.

What about the following message: "Failed to ensure PostgreSQL is running, retrying in %d seconds".